AccountDownloader/ViewModels/Controls/GroupsListItemViewModel.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AccountDownloader.Properties;
+using AccountDownloader.Services;
+using ReactiveUI;
+
+namespace AccountDownloader.ViewModels;
+
+public class GroupsListItemViewModel: ReactiveObject
+{
+    private const byte MAX_TITLE_LENGTH = 60;
+
+    public IGroup Group { get;}
+
+    public string GroupTitleLabel
+    {
+        get
+        {
+            var title  = $"{(Group.IsAdmin ? $"{Resources.GroupsAdminIndicator} " : "")}{Group.Name}";
```

Comment:
This all reads a bit cluttered due to nested evaluation. I'll clean it up after its merged.

Reply:
It is a bit cluttered. I can clean that up.